### ISSUE TITLE
External Issue

### ISSUE THREAD

**Issue by Ldennis3044** [geany/geany#1751]
Hi, I'm in a programming class and I'm using Geany to do Python work. When I tried to do the "bicycles.py" exercise, I got the following error when executing; " ' . ' is not recognized as an internal or external command, operable program or batch."

this was the list:
bicycles = ['trek', 'cannondale', 'redline', 'specialized',]
print(bicycles)

Please help!!!

**Comment by elextr** [comment]
Have you saved the file?

**Comment by Ldennis3044** [comment]
yes

**Comment by elextr** [comment]
is it just that file?  Check you havn't modified the build command settings.

**Comment by Ldennis3044** [comment]
its just this file, every other file is find. I haven't messed with the build command settings at all.

**Comment by elextr** [comment]
Weird. What OS, version of Geany?

**Comment by Ldennis3044** [comment]
I was recommended to download version 1.32

**Comment by elextr** [comment]
yeah, thats the latest, but what operating system are you running on?
